.gitlab-ci.yml

```diff
@@ -8,11 +8,23 @@ before_script:
   #- pip3 install biopython==1.76
   - wget -qO- https://get.nextflow.io | bash && cp nextflow /usr/local/bin/nextflow
   - nextflow help
-  - wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+  - wget -q https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh
   - mkdir /root/.conda
-  - bash Miniconda3-latest-Linux-x86_64.sh -b && cp /root/miniconda3/bin/* /usr/local/bin/
-  - rm -f Miniconda3-latest-Linux-x86_64.sh
-  - conda --version
+  - bash Mambaforge-Linux-x86_64.sh -b -p /usr/local/bin/conda
+  - rm -f Mambaforge-Linux-x86_64.sh
+  - source /usr/local/bin/conda/etc/profile.d/conda.sh
+  - source /usr/local/bin/conda/etc/profile.d/mamba.sh
+  - which mamba 
+  - mamba --version
+  # this is a workaround for https://github.com/mamba-org/mamba/issues/1993
+  - echo "use_lockfiles:" >> ~/.condarc
```

Comment:
this is an issue we had in LOHHLA, the file has to be `~/.mambarc`

Reply:
Thanks. I'll change it before merging